### ISSUE TITLE
issues Pickup Date not correct. Its -1 day by selecting date. 

### ISSUE THREAD

**Issue by rajhim** [valor-software/ng2-bootstrap#4995]
### Versions of ngx-bootstrap, Angular, and Bootstrap:

ngx-bootstrap: 3.1.4

Angular:6

Bootstrap:3

Pickup Date not correct. Its -1 day by selecting date. 
For eg:
<img width="428" alt="screen shot 2019-01-09 at 1 21 19 pm" src="https://user-images.githubusercontent.com/25143929/50883930-99de6800-1411-11e9-88b6-ac2885c98dd1.png">

https://stackblitz.com/edit/ngx-bootstrap-cwdcm9?file=app/app.component.html

**Comment by arangates** [comment]
yeah ,...this is because it adds time ,...you can see after T there is some junk value for a very long time
